instagram/README.md

```diff
@@ -0,0 +1,33 @@
+# Instagram Edge App
+
+## Setup
+
+To build the container, run the below command from the root directory:
+
+`docker build -t screenly/instagram -f instagram/Dockerfile .`
+
+To run the container, run:
+```sh
+docker run \
+  -e INSTAGRAM_API_TOKEN=<YOUR_INSTAGRAM_API_TOKEN> \
+  -v /app/instagram/node_modules \
```

Comment:
This isn't a valid volume mount (missing `:`), but also don't see why we would need to expose `node_modules` to the host. They should remain in the docker container.

Reply:
That's a bookmark, but yes it won't be required unless we are mounting the entire src instagram directory. Will remove it.

Reply:
In fact, maybe we just *not* include any of the code base, but instead just have the 'build environment' in Docker and just volume mount the source code. That might be cleaner.